src/context/WalletProvider/WalletConnectV2/config.ts

```diff
@@ -96,6 +96,11 @@ export const walletConnectV2ProviderConfig: EthereumProviderOptions = {
     'eth_signTransaction',
   ],
   showQrModal: true,
+  qrModalOptions: {
+    themeVariables: {
+      '--wcm-z-index': '2000',
```

Comment:
Undocumented magic css variable, thankfully we did have types 🙏🏽 default is 89 which was way too low
https://github.com/WalletConnect/modal/blob/0fb36833dbd9c896940a3c65485049e56ad5f018/packages/modal-ui/src/utils/ThemeUtil.ts#L35

Reply:
Yeah it's due to the fact we have zIndexes over 89, looks acceptable to me 👍